### PR TITLE
feat: bitgetWallet support

### PR DESCRIPTION
**Pull request by kisenyang**
Hi vu3th team,

This PR adds the bitget wallet to the list of supported wallets to make it easier for our users to connect to vue-dapp dapps.

Please let me know if anything needs to be added or changed to satisfy your requirements.

Cheers,
bitget wallet team

### CODE REVIEW

File: packages/legacy/src/connectors/bitgetWallet.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    try {
+      await provider.request({
+        method: 'wallet_switchEthereumChain',
+        params: [{ chainId: id }],
+      })
+    } catch (error: unknown) {
+      if ((<ProviderRpcError>error).code === 4902) {
+        try {
+          // await provider.request({
```
Comment (re2005) [inline packages/legacy/src/connectors/bitgetWallet.ts:128]: Comments are intended here ?

Comment (kisenyang) [reply packages/legacy/src/connectors/bitgetWallet.ts]: Thank you very much for your reply.  We have made the appropriate processing and modification and added comments to the code

**Comment by chnejohnson** [comment]
Hi, @kisenyang Thanks for your contribution. I have reviewed the code and tried to run it. Help me export the wallet connector in this [file](https://github.com/vu3th/vue-dapp/blob/main/packages/legacy/src/connectors/index.ts)

I will publish a beta version for testing the wallet, but note that I'm currently updating vue-dapp, so there may be some changes in the near future.

**Comment by kisenyang** [comment]
Thank you for your reply.  I have exported the BitgetWalletConnector and we will keep an eye on vue-dapp

**Comment by chnejohnson** [comment]
@kisenyang Thanks, I've published `vue-dapp@0.12.0-beta.7`, so now you can test the wallet with it. There have been some fixes on the [bitget-wallet](https://github.com/vu3th/vue-dapp/tree/bitget-wallet) branch, which is the codebase I published.

There's also a `vue3-dapp-starter` template using `vue-dapp@0.12.0-beta.7` on the [0.12.x](https://github.com/vu3th/vue3-dapp-starter/tree/v0.12.x) branch.

If you encounter any issues, please feel free to let me know

**Comment by chnejohnson** [comment]
Now you can build bitgetWallet as a new packages in vue-dapp. Go for it 🔥

**Pull request closed without merging**